### ISSUE TITLE
Search Landmark example has WCAG Use of Color violation

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/aria-practices#2505]
Spotted this when testing with axe. On the following page, under the "Definitions" header there's a link with the text "search landmarks". This link is only distinguishable from other by its blue color. This has a contrast of 2.7:1 to the black text. This should be at least 3:1, or maybe better yet the link should be underlined by default, instead of only on hover.

https://www.w3.org/WAI/ARIA/apg/example-index/landmarks/search.html

<img width="474" alt="Screenshot of the issue" src="https://user-images.githubusercontent.com/530687/194751790-cc8fbd9b-6110-4277-893d-258a647e1e6c.png">

**Comment by mcking65** [comment]
Fix merged to main. Plan to go to production this week.

Thank you @WilcoFiers for reporting the problem and to @jongund for help with the fix.
